source/renderer/app/components/loading/syncing-connecting/StatusIcons.js

```diff
@@ -244,29 +244,34 @@ export default class StatusIcons extends Component<Props> {
     paramName !== 'nodeState' &&
     this.props.nodeState !== CardanoNodeStates.RUNNING;
 
-  getIconWithToolTip = (icon: string, paramName: string) => (
-    <Tooltip
-      className={this.getTooltipClassname(paramName)}
+  getIconWithPopover = (icon: string, paramName: string) => (
+    <PopOver
+      offset={[0, 14]}
+      themeVariables={{
+        '--rp-pop-over-bg-color':
+          'var(--theme-loading-status-icons-tooltip-color)',
+        '--rp-pop-over-border-radius': '5px',
+        '--rp-bubble-padding': '6px 12px 7px',
+      }}
+      contentClassName={this.getTooltipClassname(paramName)}
       key={paramName}
-      themeOverrides={tooltipStyles}
-      skin={TooltipSkin}
-      tip={this.getTip(paramName, this.props[paramName])}
+      content={this.getTip(paramName, this.props[paramName])}
     >
       <button className={styles.iconButton} onClick={this.props.onIconClick}>
         <SVGInline svg={icon} className={this.getClassName(paramName)} />
       </button>
-    </Tooltip>
+    </PopOver>
   );
 
   render() {
     return (
       <div className={styles.component}>
         {[
-          this.getIconWithToolTip(nodeStateIcon, 'nodeState'),
-          this.getIconWithToolTip(isNodeRespondingIcon, 'isNodeResponding'),
-          // this.getIconWithToolTip(isNodeSubscribedIcon, 'isNodeSubscribed'),
-          this.getIconWithToolTip(isNodeTimeCorrectIcon, 'isNodeTimeCorrect'),
-          this.getIconWithToolTip(isNodeSyncingIcon, 'isNodeSyncing'),
+          this.getIconWithPopover(nodeStateIcon, 'nodeState'),
+          this.getIconWithPopover(isNodeRespondingIcon, 'isNodeResponding'),
+          // this.getIconWithPopover(isNodeSubscribedIcon, 'isNodeSubscribed'),
```

Comment:
Do we need this commented line? @DominikGuzei

Reply:
It was already there before (i just updated the function name), I'm not sure if we need it - but that's out of scope of this PR